cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,85 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should handle pagination', () => {
+    cy.get('.-next').should('exist');
+
+    cy.get('.-previous').should('exist');
+  });
+
+  it('should handle rows count selection', () => {
+    cy.get('select[aria-label="rows per page"]')
+      .should('exist')
+      .select('5 rows');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('have.value', '5');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('exist')
+      .select('20 rows');
+
+    cy.get('select[aria-label="rows per page"]')
+      .should('have.value', '20');
+  });
+
+  it('should add a new worker', () => {
```

Comment:
will be perfect if you create a user before that test instead of using the hardcoded values